array.libsonnet

```diff
@@ -0,0 +1,37 @@
+local d = import 'doc-util/main.libsonnet';
+
+{
+  '#': d.pkg(
+    name='array',
+    url='github.com/jsonnet-libs/xtd/array.libsonnet',
+    help='`array` implements helper functions for processing arrays.',
+  ),
+
+  '#slice':: d.fn(
+    '`slice` works the same as `std.slice` but with support for negative index/end.',
```

Comment:
I've got a PR open to upstream this: https://github.com/google/jsonnet/pull/1093